src/components/jobPosting/jobPosting.module.css

```diff
@@ -0,0 +1,50 @@
+.jobPosting {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  border-radius: 24px;
+  padding: 8px 24px;
+  gap: 5rem;
+  width: 100%;
+  background-color: var(--text-primary-light);
+  color: var(--text-primary);
+}
+
+.details {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 0.5rem;
+  width: 100%;
+}
+.jobPosting::after {
+  justify-self: end;
+  width: 1.5rem;
+  height: 1.5rem;
+  -webkit-mask: url("/_assets/arrow-right.svg") no-repeat center;
+  -webkit-mask-size: cover;
+  mask: url("/_assets/arrow-right.svg") no-repeat center;
+  mask-size: cover;
+  background-color: var(--background-bg-dark);
+}
+
+.role {
+  width: 250px;
+  @media (max-width: 640px) {
+    width: 100%;
+  }
+}
+
+.locations {
+  padding: 6px 12px;
```

Comment:
minor: bytte ut med rem

Reply:
Fixed in https://github.com/varianter/variant.no/pull/917/commits/22dc8c45833b54719618d80bcf4bd88dbee7df45